api/symphony-api.json

```diff
@@ -13,17 +13,23 @@
       "lng": "-122.12826"
     },
     "currentSite": {
-      "baseUrl": "http://symphony-service:8080/v1alpha2/",
+      "baseUrl": "https://symphony-service:8081/v1alpha2/",
       "username": "admin",
       "password": ""
-    }
+    }  
   },
   "api": {
     "pubsub": {
       "shared": true,
       "provider": {
-        "type": "providers.pubsub.memory",
-        "config": {}
+        "type": "providers.pubsub.redis",
```

Comment:
Why you change this?

Reply:
This file is mis-replaced. will recover back